lib/components/narrative/trip-tools.js

```diff
@@ -105,8 +114,20 @@ class CopyUrlButton extends Component {
           onClick={this._onClick}
         >
           {this.state.showCopied
-            ? <span><i className='fa fa-check' /> Copied</span>
-            : <span><i className='fa fa-clipboard' /> Copy Link</span>
+            ? (
+              <span>
+                <i className='fa fa-check' />
+                {' '}
```

Comment:
I think adding whitespace is a no-no when using internationalization. If this is for styling purposes to add padding or something, the styling should be changed instead.